src/pipeline/eval.py

```diff
@@ -4,11 +4,13 @@
 from string import Template
 from datetime import datetime
 from datasets import load_dataset, DatasetDict
+from collections import deque
 
 from ..gen.gemini import get_model as get_service_model
 from ..gen.utils import call_service_llm
 
 JSON_KEYS_TO_CHECK = ["similarity_assessment.score", "precision_assessment.score"]
+RATE_LIMIT_PER_MINUTE = 60 # Gemini-1.0 ratelimit
```

Comment:
maybe this constant is better to be configurable via CLI

Reply:
You are not supposed to configure the rate limit. It's a constant that's set by an organization and not by a user.

Reply:
The rate limit changes over time, and it is different from organization to organization, from paid users to free users. If we consider to embrace other APIs besides Gemini, this number would vary a lot.

I also saw this number could be a sort of price limiter.